src/worker.js

```diff
@@ -6,6 +6,7 @@ import Path from 'path'
 import { create } from 'process-communication'
 import { FindCache, findCached } from 'atom-linter'
 import * as Helpers from './worker-helpers'
+import { isConfigAtHomeRoot } from './helpers'
```

Comment:
`helpers` brings in a _lot_ of things, and isn't intended to be used from the worker process (gotta love our convoluted design here...). Maybe this function should live in `worker-helpers` and helpers imports it from there? Either way is messy 😕.